Current docker image lacks needed dynamic libraries

Due to the OpenSSL dependency of this program, turns out that the resulting binary is dynamic linked:
```
# file nrpe_exporter
nrpe_exporter: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=fa46de95fb47fab8f7534ba7fa624f4158b1d6cc, for GNU/Linux 3.2.0, not stripped
# ldd nrpe_exporter
	linux-vdso.so.1 (0x00007fffdd794000)
	libssl.so.1.1 => /usr/lib/libssl.so.1.1 (0x00007f6196ef8000)
	libcrypto.so.1.1 => /usr/lib/libcrypto.so.1.1 (0x00007f6196c18000)
	libpthread.so.0 => /usr/lib/libpthread.so.0 (0x00007f6196bf0000)
	libc.so.6 => /usr/lib/libc.so.6 (0x00007f6196a28000)
	libdl.so.2 => /usr/lib/libdl.so.2 (0x00007f6196a20000)
	/lib64/ld-linux-x86-64.so.2 => /usr/lib64/ld-linux-x86-64.so.2 (0x00007f6196ff8000)
```

Running without CGO doesn't seem to be an option here, sadly:
```
# CGO_ENABLED=0 go build nrpe_exporter.go
# github.com/spacemonkeygo/openssl
../../../.go/pkg/mod/github.com/spacemonkeygo/openssl@v0.0.0-20181017203307-c2dcc5cca94a/http.go:38:14: undefined: NewCtxFromFiles
../../../.go/pkg/mod/github.com/spacemonkeygo/openssl@v0.0.0-20181017203307-c2dcc5cca94a/net.go:24:7: undefined: Ctx
../../../.go/pkg/mod/github.com/spacemonkeygo/openssl@v0.0.0-20181017203307-c2dcc5cca94a/net.go:32:16: undefined: Server
../../../.go/pkg/mod/github.com/spacemonkeygo/openssl@v0.0.0-20181017203307-c2dcc5cca94a/net.go:43:43: undefined: Ctx
../../../.go/pkg/mod/github.com/spacemonkeygo/openssl@v0.0.0-20181017203307-c2dcc5cca94a/net.go:51:41: undefined: Ctx
../../../.go/pkg/mod/github.com/spacemonkeygo/openssl@v0.0.0-20181017203307-c2dcc5cca94a/net.go:79:38: undefined: Ctx
../../../.go/pkg/mod/github.com/spacemonkeygo/openssl@v0.0.0-20181017203307-c2dcc5cca94a/net.go:79:62: undefined: Conn
../../../.go/pkg/mod/github.com/spacemonkeygo/openssl@v0.0.0-20181017203307-c2dcc5cca94a/net.go:96:45: undefined: Ctx
../../../.go/pkg/mod/github.com/spacemonkeygo/openssl@v0.0.0-20181017203307-c2dcc5cca94a/net.go:97:20: undefined: Conn
```

and trying to build it statically should be possible, however:
```
# go build -ldflags="-extldflags '-static'" nrpe_exporter.go 
# command-line-arguments
/usr/lib/go/pkg/tool/linux_amd64/link: running gcc failed: exit status 1
/usr/bin/ld: cannot find -lssl
/usr/bin/ld: cannot find -lcrypto
/usr/bin/ld: /tmp/go-link-932140215/000004.o: in function `_cgo_26061493d47f_C2func_getaddrinfo':
/tmp/go-build/cgo-gcc-prolog:58: warning: Using 'getaddrinfo' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
collect2: error: ld returned 1 exit status
```
I really don't want to wrap my head around linker issues atm, so using another base image should suffice for now.

Since the current used base image is based on busybox, which only includes the static binaries for busybox itself with no other shared libraries in the system, the image fails as seen in #13.

I'll try to do a PR with the needed changes to make it run in an easier way.

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,8 @@
-FROM quay.io/prometheus/busybox:latest
+FROM alpine:latest
+
+RUN apk update && apk add --no-cache openssl
+
+RUN mkdir /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
 
 COPY nrpe_exporter  /bin/nrpe_exporter
 

